src/libraries/Native/Unix/System.IO.Ports.Native/pal_serial.c

```diff
@@ -44,3 +52,386 @@ int SystemIoPortsNative_SerialPortClose(intptr_t handle)
     ioctl(fd, TIOCNXCL);
     return close(fd);
 }
+
+int32_t ConvertErrorPlatformToPal(int32_t platformErrno)
+{
+    switch (platformErrno)
+    {
+        case 0:
+            return Error_SUCCESS;
```

Comment:
The managed counter-part (`Unix\Interop.Errors.cs` and friends) is shared between different shims. Can we have this in some shared file as well to avoid duplication?